core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -352,7 +356,7 @@ class AuthorizerIntegrationTest extends KafkaServerTestHarness {
   }
 
   @Test
-  def testConsumeWithNoTopicAccess() {
+  def testConsumeWithoutTopicDescribeAccess() {
```

Comment:
Hi 
sorry but could you confirm ... :
If `ConsumerCoordinator.OffsetFetchResponseHandler` raises an UTOP 
that is a retriable exception and `ConsumerCoordinator.fetchCommittedOffsets` remains stuck in the `while` loop.
The client would not see the exception. The unit test would have to be coded for a timeout. 
Is this the behavior you'd like to see ? With a log entry while stuck in the loop ?